src/schema/payment.ts

```diff
@@ -25,15 +22,9 @@ export interface AuthRequestPostPayment {
   context?: PaymentContext
 }
 
-type PeriodType =
-  | "Day"
-  | "Week"
-  | "Fortnight"
-  | "Month"
-  | "Half-year"
-  | "Year"
+type PeriodType = "Day" | "Week" | "Fortnight" | "Month" | "Half-year" | "Year";
```

Comment:
Want to try and make sure your formatter doesn't add semicolons to the end. Its not part of our style. Can see that guide here: https://www.notion.so/moneyhub/Development-process-and-style-guide-081e887009414e88a9fe5e8e3df1bf8a#9febd1cc3e0a4a45ad11c873dd86cc54

Reply:
Removed semicolons now, formatter was to blame

Reply:
It's all reformatted actually, I reckon format on save with prettier installed globally? If so, I think [this](https://www.notion.so/moneyhub/Development-process-and-style-guide-081e887009414e88a9fe5e8e3df1bf8a?pvs=4#9febd1cc3e0a4a45ad11c873dd86cc54) might be useful, though not familiar with the setup in this particular repo